Bio/SeqUtils/__init__.py

```diff
@@ -467,6 +468,69 @@ def six_frame_translations(seq, genetic_code=1):
     return res
 
 
+def count_kmers(seq, k, circular=False, ordered=True):
```

Comment:
It is clear from your examples that you are thinking of protein sequences here.

However, kmer analysis is common with nucleotides, where odd kmers are preferred since their reverse complement is different. It would at least be worth adding something to the docstring saying it does not consider the reverse complent?